models/feedItem.js

```diff
@@ -0,0 +1,18 @@
+var mongoose = require( 'mongoose' );
+
+// create feedItem schema
+var options = { discriminatorKey: 'type' };
+
+var feedItemSchema = new mongoose.Schema({
+    _id: {
+        $oid: String
+    },
+    title: String,
+    description: String,
+    index: Number
+}, options);
+
+// create model
+var FeedItem = mongoose.model('feeditems', feedItemSchema);
```

Comment:
What's the rule for capitalization? I see `FeedItem`, `feeditems`, and `feedItemSchema`. When we pass around values as string literals (I'm looking at you `'feeditems'`) I think that's kind of important to standardize on. Thanks a lot, javascript.